src/Goerli-DssSpellCollateral.sol

```diff
@@ -75,174 +37,114 @@ contract DssSpellCollateralAction {
-        address MCD_PSM_USDC_A
+    uint256 internal constant WAD    = 10 ** 18;
+    uint256 internal constant MILLION = 10 ** 6;
+    uint256 internal constant RAY = 10 ** 27;
+    uint256 internal constant RAD = 10 ** 45;
+
+    // Change clip parameters
+    function collateralAuctionParameterChanges(
+        address MCD_DOG
     ) internal {
-        // RWA007-A collateral deploy
-        bytes32 ilk      = "RWA007-A";
-        uint256 decimals = GemAbstract(RWA007).decimals();
-
-        // Sanity checks
-        require(GemJoinAbstract(MCD_JOIN_RWA007_A).vat()                             == MCD_VAT,                                    "join-vat-not-match");
-        require(GemJoinAbstract(MCD_JOIN_RWA007_A).ilk()                             == ilk,                                        "join-ilk-not-match");
-        require(GemJoinAbstract(MCD_JOIN_RWA007_A).gem()                             == RWA007,                                     "join-gem-not-match");
-        require(GemJoinAbstract(MCD_JOIN_RWA007_A).dec()                             == decimals,                                   "join-dec-not-match");
-
-        require(RwaUrnLike(RWA007_A_URN).vat()                                       == MCD_VAT,                                    "urn-vat-not-match");
-        require(RwaUrnLike(RWA007_A_URN).jug()                                       == MCD_JUG,                                    "urn-jug-not-match");
-        require(RwaUrnLike(RWA007_A_URN).daiJoin()                                   == MCD_JOIN_DAI,                               "urn-daijoin-not-match");
-        require(RwaUrnLike(RWA007_A_URN).gemJoin()                                   == MCD_JOIN_RWA007_A,                          "urn-gemjoin-not-match");
-        require(RwaUrnLike(RWA007_A_URN).outputConduit()                             == RWA007_A_OUTPUT_CONDUIT,                    "urn-outputconduit-not-match");
         
-        require(RwaJarLike(RWA007_A_JAR).chainlog()                                  == DssExecLib.LOG,                             "jar-chainlog-not-match");
-        require(RwaJarLike(RWA007_A_JAR).dai()                                       == DssExecLib.dai(),                           "jar-dai-not-match");
-        require(RwaJarLike(RWA007_A_JAR).daiJoin()                                   == MCD_JOIN_DAI,                               "jar-daijoin-not-match");
-
-        require(RwaOutputConduitLike(RWA007_A_OUTPUT_CONDUIT).dai()                  == DssExecLib.dai(),                           "output-conduit-dai-not-match");
-        require(RwaOutputConduitLike(RWA007_A_OUTPUT_CONDUIT).gem()                  == DssExecLib.getChangelogAddress("USDC"),     "output-conduit-gem-not-match");
-        require(RwaOutputConduitLike(RWA007_A_OUTPUT_CONDUIT).psm()                  == MCD_PSM_USDC_A,                             "output-conduit-psm-not-match");
+        // Load Clippers for ilks being modified
```

Comment:
can you link to the poll and vote for these (or if they are separate, the for each group)